README.md

```diff
@@ -1,2 +1,100 @@
+2. Install the dependencies using `npm install`
+3. Run the `parallel_test.js` script using `node parallel_test.js`
+
+## Run sample test on privately hosted websites
+
+1. You have to download the BrowserStack Local binary from the links below (depending on your environment):
+   * [OS X (10.7 and above)](https://www.browserstack.com/browserstack-local/BrowserStackLocal-darwin-x64.zip)
+   * [Linux 32-bit](https://www.browserstack.com/browserstack-local/BrowserStackLocal-linux-ia32.zip)
+   * [Linux 64-bit](https://www.browserstack.com/browserstack-local/BrowserStackLocal-linux-x64.zip)
+   * [Windows (XP and above)](https://www.browserstack.com/browserstack-local/BrowserStackLocal-win32.zip)
+2. Once you have downloaded and unzipped the file, you can initiate the binary by running the command: `./BrowserStackLocal --key YOUR_ACCESS_KEY`
+3. Once you see the terminal say “\[SUCCESS\] You can now access your local server(s) in our remote browser”, your local testing connection is considered established.
+4. You can then run the sample Local test using `node local_test.js`
+
+## Supported Browsers and Playwright versions
+
+BrowserStack Playwright tests in beta supports the following browsers across the following OS versions:
+
+### Browsers supported
+1. Bundled Chromium with Playwright (`'browser': 'playwright-chromium'`)
+2. Bundled Firefox with Playwright (`'browser': 'playwright-firefox'`)
+3. Bundled Webkit with Playwright (`'browser': 'playwright-webkit'`)
+4. Branded Google Chrome (`'browser': 'chrome'`)
+   * You can test on multiple versions of branded Google Chrome using another capability `'browser_version': 'latest'`. Supported options are `latest`, `latest-beta`, `latest-1` and so on.
+   * We would be supporting Chrome v83 and above. You can also specify `browser_version` as `88` or `91` and so on.
+5. Branded Microsoft Edge (`'browser': 'edge'`)
+   * You can test on multiple versions of branded Microsoft Edge using another capability `'browser_version': 'latest'`. Supported options are `latest`, `latest-beta`, `latest-1` and so on.
+   * We would be supporting Edge v83 and above. You can also specify `browser_version` as `88` or `91` and so on.
+
+**The bundled browser version that will be used to run your test is the same that comes bundled with the Playwright version**. Supported Playwright versions and the option to specify them is given in a later section of this page.
+
+**Note**: You can also run your tests in any of the mobile `devices` that Playwright supports for emulation. You can find the [sample code for running in an iPhone emulator](https://github.com/sourav-kundu/playwright-browserstack/blob/master/sample_test_on_iPhone.js) and the [sample for running on Pixel](https://github.com/sourav-kundu/playwright-browserstack/blob/master/sample_test_on_Pixel.js). The complete list of `devices` supported by Playwright can be found [here](https://github.com/microsoft/playwright/blob/master/src/server/deviceDescriptors.js)
```

Comment:
Do we need to change `https://github.com/sourav-kundu/playwright-browserstack/blob/master/sample_test_on_iPhone.js` from sourav-kundu to browserstack?

Reply:
changed